app/section/sections/browser-additional-information/ui.test.js

```diff
@@ -0,0 +1,181 @@
+};
+
+fixture.only('Show Solution Description page')
+  .afterEach(async (t) => {
+    const isDone = nock.isDone();
+    if (!isDone) {
+      nock.cleanAll();
+    }
+
+    await t.expect(isDone).ok('Not all nock interceptors were used!');
+  });
+
+test('should render the Solution Description page title', async (t) => {
+  await pageSetup(t);
+
+  const title = Selector('[data-test-id="section-title"]');
+
+  await t
+    .expect(title.innerText).eql('Browser based - additional information');
+});
+
+test('should render main advice of section', async (t) => {
+  await pageSetup(t);
+
+  const mainAdvice = Selector('[data-test-id="section-main-advice"]');
+
+  await t
+    .expect(mainAdvice.innerText).eql('Answer the following questions about your browser based solution');
+});
+
+test('should render all the advice of the section', async (t) => {
+  await pageSetup(t);
+
+  const sectionManifest = new ManifestProvider().getSectionManifest('solution-description');
+
+  const additionalAdvice = Selector('[data-test-id="section-additional-advice"]');
+
+  await t
+    .expect(additionalAdvice.innerText).eql('This information is diplayed on the Marketing page for buyers to view');
+});
+
+test('should render the solution summary question', async (t) => {
+  await pageSetup(t);
+
+  const summaryQuestion = Selector('[data-test-id="question-additional-information"]');
+
+  await t
+    .expect(summaryQuestion.find('label.nhsuk-label').innerText).eql('Do you have any additional browser based information (optional)')
+    .expect(summaryQuestion.find('span.nhsuk-hint').innerText).eql('Add any additional information or requirement your Solution needs to function')
+    .expect(summaryQuestion.find('textarea').count).eql(1)
+    .expect(summaryQuestion.find('[data-test-id="textarea-field-footer"]').innerText).eql('You have 500 characters remaining');
+});
+
+test('should populate the text fields with existing data', async (t) => {
+  pageSetup(t, true);
+
+  const theQuestions = Selector('form');
+
+  await t
+    .expect(theQuestions.find('[data-test-id="question-additional-information"]').find('textarea').value).eql('The solution additional information')
+});
+
+test('should render the submit button', async (t) => {
+  await pageSetup(t);
+
+  const submitButton = Selector('[data-test-id="section-submit-button"]');
+
+  await t
+    .expect(submitButton.find('button').count).eql(1);
+});
+
+test('should show error summary and validation for questions when they exceed the maxLength', async (t) => {
+  await pageSetup(t);
+
+  nock('http://localhost:8080')
+    .put('/api/v1/Solutions/S100000-001/sections/browser-additional-information')
+    .reply(400, {
+      maxLength: ['additional-information'],
+    });
+
+  const oneHundredCharacters = '0123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789';
+  const thousandCharacters = oneHundredCharacters.repeat(10);

```

Comment:
actually I don't think we need this. The mock will return the error regardless of what we put in that text-area